src/stories/availability-label/AvailabilityLabel.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { AvailabilityLabel } from "./AvailabilityLabel";
+
+export default {
+  title: "Components / Availability Label",
+  component: AvailabilityLabel,
+  decorators: [withDesign],
+  argTypes: {},
+  parameters: {},
+} as ComponentMeta<typeof AvailabilityLabel>;
+
+const Template: ComponentStory<typeof AvailabilityLabel> = (args) => (
+  <AvailabilityLabel {...args} />
+);
+
+export const Available = Template.bind({});
+Available.args = {
+  manifestation: "bog",
+  availability: "Hjemme",
+  status: "available",
+  pagefoldProps: {
+    inherit: false,
+    container: false,
+    size: "xsmall",
+    classes: "availability-label availability-label--unselected text-label",
+    colorClass: "success",
```

Comment:
These seem to be duplicated from `getPageFoldProps()` where they are derived from `status`. Do they have to be defined as a part of the story?

The same goes for the next two stories.

Reply:
You are completely right! We don't actually need these.